server.js

```diff
@@ -1101,8 +1101,9 @@ cache(function(data, match, sendBadge, request) {
     }
     try {
       var data = JSON.parse(buffer);
-      var receiving = data.receiving || data.taking;
-      if (receiving) {
+      // Avoid falsey checks because amounts may be 0
+      var receiving = isNaN(data.receiving) ? data.taking : data.receiving;
```

Comment:
I went with ``isNan`` rather than a standard null check, since it's just ever-so-slightly more future-proof. If Gratipay ever starts actually returning a literal "annonymous" string for receiving/taking in the future, this won't need to be updated.

Not that that's particularly likely to happen or worth going out the way to do, but since ``isNan`` is just as concise/readable as the alternative, I figured "why not?"